app/src/main/resources/logback.xml

```diff
@@ -84,7 +80,7 @@
     <logger name="io.netty" level="WARN"/>
     <logger name="org.asynchttpclient" level="WARN"/>
 
-    <root>
+    <root level="${EXPLORER_LOG_LEVEL:-INFO}">
```

Comment:
Does this change the log level of `ERRFILE` too?

Reply:
no because the `ERRFILE` has a `Threshold` : https://github.com/alephium/explorer-backend/blob/cda9dae28be07a06c7274bd221d1edb840ddaa9f/app/src/main/resources/logback.xml#L16-L18